BruteShark/PcapAnalyzerTest/PcapAnalyzerTest.csproj

```diff
@@ -15,6 +15,17 @@
 
 	<ItemGroup>
 	  <ProjectReference Include="..\PcapAnalyzer\PcapAnalyzer.csproj" />
-	</ItemGroup>
+  </ItemGroup>
+
+
+  <ItemGroup>
```

Comment:
Since its a small Json I think  it better to add that as a string variable (hate csproj complex files 😸 )

Reply:
i had some issues with that json string because of the " in it but i'm sure they are possible to overcome